plugins/newsdownloader.koplugin/main.lua

```diff
@@ -17,9 +17,22 @@ local NewsDownloader = WidgetContainer:new{}
 local initialized = false  -- for only once lazy initialization
 local FEED_CONFIG_FILE = "feeds.xml"
 local FILE_EXTENSION = ".html"
-local FEED_SOURCE_SUFFIX = "_rss_tmp.xml"
+local NEWS_DL_DIR_NAME = "news"
```

Comment:
no, it's just a global constant in the code right now. i have plan to move it to its own setting file and make it editing from the UI.